Explorer/Assets/Scripts/CrdtEcsBridge/JsModulesImplementation/EngineAPIImplementation.cs

```diff
@@ -41,6 +43,8 @@ public class EngineAPIImplementation : IEngineApi
         private readonly CustomSampler worldSyncBufferSampler;
         private bool isDisposing;
 
+        public HashSet<ProcessedCRDTMessage> OutgoingCRDTMessages { get; } = new ();
```

Comment:
I think we should not store and expose messages like this as it can break management of their life-cycle that is very hard to debug: I'd propose to call into `SDKObservableEventsEngineApiWrapper` and process messages here straight-away and if needed create a memory copy there and let it manage the lifecycle of its copy in separation. Thus, we don't break this class.